wandb/proto/wandb_settings.proto

```diff
@@ -23,6 +23,12 @@ message OpenMetricsFilters {
   }
 }
 
+message RunMoment {
```

Comment:
add docs to the new message and its fields

Reply:
done